arduino-ide-extension/src/common/protocol/sketches-service.ts

```diff
@@ -88,7 +88,8 @@ export namespace Sketch {
         export const MAIN = ['.ino', '.pde'];
         export const SOURCE = ['.c', '.cpp', '.s'];
         export const ADDITIONAL = ['.h', '.c', '.hpp', '.hh', '.cpp', '.s', '.json'];
-        export const ALL = Array.from(new Set([...MAIN, ...SOURCE, ...ADDITIONAL]));
+        export const DOCUMENTATION = ['.md', '.adoc'];
```

Comment:
Please add them to _additional_. There is no _documentation_ category on the CLI. See [here](https://github.com/arduino/arduino-cli/blob/c4c723eb990accbd810ced70f7d14935dad831e1/arduino/globals/globals.go#L32-L42). [Here](https://github.com/arduino/arduino-cli/pull/1235) is the PR for the CLI. Thank you!